source/xml/reading.md

```diff
@@ -376,6 +377,7 @@ their respective XML elements:
             }
             return $books;
         },
+        // handle a single book
         '{http://example.org/books}book' => function($reader) {
             $book = new Book();
```

Comment:
I agree, I think it's a great idea. This whole conversation actually makes me thing that most of the things in should actually just be a couple of deserializer functions instead. It makes so much more sense for the general case.

So I'm thinking, after you added this function I will create a few more functions just like it. Then in the next major (2.0) release I'll remove most/all of the Element classes.